### PR TITLE
gdalsource

### PR DESCRIPTION
**Pull request by michaelmattig**

### CODE REVIEW

**Comment by michaelmattig** [comment]
[Template anhand der GDALSource](https://trello.com/c/37YxDz28/404-template-anhand-der-gdalsource)

File: src/operators/gdalsource.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# GdalSource
+
+The `GdalSource` is a _source operator_ that reads raster data using GDAL.
+The counter part for vector data is the `OgrSource`.
+
+## Parameters
+
+| Parameter | Type        | Description                        | Example Value | Default Value |
+| --------- | ----------- | ---------------------------------- | ------------- | ------------- |
+| `dataset` | `DatasetId` | The id of the dataset to be loaded | <code>"{<br>&nbsp;&nbsp;"type": "internal",<br>&nbsp;&nbsp;"datasetId": "a626c880-1c41-489b-9e19-9596d129859c"<br>}</code> | |
```
Comment (ChristianBeilschmidt) [inline src/operators/gdalsource.md:10]: Maybe put a `<pre>` around to get a nicer output of the example value.

File: src/operators/gdalsource.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
+# GdalSource
+
+The `GdalSource` is a _source operator_ that reads raster data using GDAL.
+The counter part for vector data is the `OgrSource`.
```
Comment (ChristianBeilschmidt) [inline src/operators/gdalsource.md:4]: You could link to the `OgrSource`.

Moreover, could you replace the currently empty pages with placeholders?
From https://rust-lang.github.io/mdBook/format/summary.html you can create a draft chapter by just linking to no page.

It is not really part of this PR, but otherwise you link to all empty pages.

File: src/operators/gdalsource.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+{
+  "type": "GdalSource",
+  "params": {
+    "dataset": {
+      "type": "internal",
+      "datasetId": "a626c880-1c41-489b-9e19-9596d129859c"
+    }
+  }
+}
+```
```
Comment (ChristianBeilschmidt) [inline src/operators/gdalsource.md:32]: maybe also link an external dataset?

File: src/SUMMARY.md [hunk 4]
```diff
@@ -7,7 +7,9 @@
   - [Engine](./geo-engine/engine.md)
   - [Workflows](./geo-engine/workflows.md)
 
-- [Operators](./operators.md)
+- [Operators](./operators/intro.md)
```
Comment (ChristianBeilschmidt) [inline src/SUMMARY.md:10]: Could you say 1-2 sentences as an intro to operators?

**Review by ChristianBeilschmidt — CHANGES_REQUESTED** [review]

File: src/operators/gdalsource.md (hunk 1 above)
Comment (michaelmattig) [reply src/operators/gdalsource.md]: 79b36d69847bd722e348825fe0903e7cb05ee27d

File: src/operators/gdalsource.md (hunk 2 above)
Comment (michaelmattig) [reply src/operators/gdalsource.md]: 79b36d69847bd722e348825fe0903e7cb05ee27d

File: src/operators/gdalsource.md (hunk 3 above)
Comment (michaelmattig) [reply src/operators/gdalsource.md:32]: no, this page describes the gdal source, we should explain how datasets and dataset ids work on a different page

File: src/SUMMARY.md (hunk 4 above)
Comment (michaelmattig) [reply src/SUMMARY.md:10]: 79b36d69847bd722e348825fe0903e7cb05ee27d

**Comment by ChristianBeilschmidt** [comment]
I guess we need a file with a temporary small intro, since currently the root points to 404 no document found.
The rest LGTM.

**Review by ChristianBeilschmidt — APPROVED** [review]
